contrib/.env.sample

```diff
@@ -1,9 +1,12 @@
 ALLOWED_HOSTS=*
-DEBUG=True
+CACHE_BACKEND=django.core.cache.backends.memcached.MemcachedCache
+ENVIRONMENT=development
 HTTPS_METHOD=redirect
+LOG_LEVEL=debug
 SECRET_KEY=my-secret
 USE_X_FORWARDED_HOST=False
-CACHE_BACKEND=django.core.cache.backends.memcached.MemcachedCache
+WEB_TIMEOUT=60
+WEB_WORKERS=2
```

Comment:
Can we add these new variables (and remove deprecated ones) to [Jarba's settings](https://github.com/okfn-brasil/serenata-de-amor/tree/master/jarbas#settings) docs?